Analyses/src/PointerCheck_module.cc

```diff
@@ -45,42 +45,50 @@ namespace mu2e {
       using Name=fhicl::Name;
       using Comment=fhicl::Comment;
 
-      fhicl::OptionalAtom<bool> skipDereference{Name("skipDereference"), 
-	  Comment("skip dereferencing pointers")};
-      fhicl::OptionalAtom<int> verbose{Name("verbose"), 
-	  Comment("verbose flag, 0 to 10")};
+      fhicl::Atom<bool> skipDereference{Name("skipDereference"), 
+	  Comment("skip dereferencing pointers check"),false};
+      fhicl::Atom<int> verbose{Name("verbose"), 
+	  Comment("verbose flag, 0 to 10"),1};
```

Comment:
This line sets verbose=1 by default.   It used to be zero.  Is this change intentional?